application/src/main/java/org/opentripplanner/apis/transmodel/mapping/RelativeDirectionMapper.java

```diff
@@ -1,6 +1,6 @@
 package org.opentripplanner.apis.transmodel.mapping;
 
-import org.opentripplanner.model.plan.RelativeDirection;
+import org.opentripplanner.model.plan.walkstep.RelativeDirection;
```

Comment:
Should walkstep be inside the leg package since walksteps are always part of a leg?

Reply:
I considered it, but fell down on that is is independent of leg. I do not think there is a right or wrong in this case. We can move it if you find the model more intuitive that way.